Frontend/src/components/Queue/QueueContainer.jsx

```diff
@@ -69,6 +72,7 @@ const QueueContainer = () => {
 
   return (
     <Queue
```

Comment:
I'm not sure if it makes sense to put the loader inside the Queue component directly.  I would think you would want to choose to show the queue or show the loader inside of the QueueContainer. something like this would work:

```
return(
{!loading && <Queue ... >}
{loading && <Loader ... />}
)
```